app/pages/lab/workflow.cjsx

```diff
@@ -661,6 +661,10 @@ EditWorkflowPage = React.createClass
     if confirmed
       @setState deletionInProgress: true
 
+      if @props.workflow.id is @props.project.configuration?.default_workflow
+        @props.project.update 'configuration.default_workflow': null
+        @props.project.save()
```

Comment:
Would `delete @props.project.configuration.default_workflow` be better than setting the value to null?